.circleci/config.yml

```diff
@@ -81,6 +81,9 @@ aliases:
 
 workflows:
   test_and_release:
+    when:
+      not:
+        is: pull_request  # Exclude pull request events
```

Comment:
This would break CI on `develop` and `master` (which would break releases)

Reply:
We may also want to allow runs from branches that are manually triggered (not especially important to me but @HowardBraham mentioned this idea in Slack, some people mind find this useful)

Reply:
Can we look into doing it this way as well?
https://discuss.circleci.com/t/product-launch-override-only-build-prs-setting/47807

Reply:
@Gudahtt I expanded the script to include develop and master in this commit [e3fcfc49](https://github.com/MetaMask/metamask-extension/pull/25489/commits/e3fcfc49d0eabf278d892398bdffd1d4a48f3ba9)

Reply:
@Gudahtt 
> We may also want to allow runs from branches that are manually triggered (not especially important to me but @HowardBraham mentioned this idea in Slack, some people mind find this useful)

Can we target in a different ticket, and use an agreed scheme like `trigger-CI-feature-123-test` where we always have `trigger-CI` as prefix for branches who needs to have a pipeline before creating a PR?

Reply:
@HowardBraham Thanks for the information, but I was a bit lost there. Can you elaborate more on this propsal? 
my understanding is, in the thread an API is suggested to config the setting. But we can use a schema in yml file to target the branch segmentation. wdyt?

Reply:
Yeah, I agree it's confusing and I definitely don't completely understand it either.  Basically there exist options in the API that are not available in the YML.

I thought about that `trigger-CI` thing, yeah can we do that for now?

Reply:
can you explain to me again about the `trigger-CI` thing, is this related to triggering job manually in circle ci?

Reply:
I'm not sure there's a way to configure the manual trigger thing in `config.yml`, I think the only way to do it is with that CircleCI API.

But if you make the pattern `/^(master|develop|trigger-CI-.*)$/` it should at least run automatically on those branches.

Also it's a little hard to test, but I don't think it's going to run on `master` or `develop` as written.  I think it will only run on Pull Requests.

Reply:
Thanks Howard for suggestions, i configured to trigger it in each run and integrated the pattern you suggested~